src/main/resources/alma/common/fields.xml

```diff
@@ -751,5 +751,68 @@
     </entity>
   </entity>
 
+  <!-- TODO: inCollection: missing edoweb, rpb, publisso, also type-element -->
+  <entity name="inCollection[]" flushWith="record">
+    <!-- ZDB -->
+    <entity name="" sameEntity="true" reset="true">
+      <data name="label" source="@zdbId">
+        <constant value="Zeitschriftendatenbank (ZDB)" />
+      </data>
+      <data name="id" source="@zdbId">
+        <constant value="http://www.lobid.org/resources/HT014846970#!" />
```

Comment:
Use the generic URL without 'www'.

Reply:
Did that with the new last commit.

Reply:
fixed this with the last commit

Reply:
Hm, there are still many links starting with 'www.lobid.org' left.

Reply:
+1